### PR TITLE
Documentation for griddy integration

### PR DESCRIPTION
**Pull request by bdraco**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/32591
- Link to parent pull request in the Brands repo: https://github.com/home-assistant/brands/pull/48
- This PR fixes or closes issue:

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/griddy.markdown [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: Griddy
+description: Instructions on how to integrate griddy prices into Home Assistant.
+logo: griddy.png
```
Comment (frenck) [inline source/_integrations/griddy.markdown]: Our logo's have moved to a different repository.

<https://github.com/home-assistant/brands>

Please open up a PR on that repository, adding the logo's.
Next, link it up with this PR and remove the `logo` frontmatter and file.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @bdraco!

Left one small review comment. Could you please take a look? Thanks!

File: source/_integrations/griddy.markdown (hunk 1 above)
Comment (bdraco) [reply source/_integrations/griddy.markdown]: Hopefully I got this right :)

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Pull request merged**